### ISSUE TITLE
Infinite deposits

### ISSUE THREAD

**Issue by Wuzzy2** [unknown-horizons/unknown-horizons#2927]
In UH 2019.1, it seems all deposits are infinite, is this intentional?

How to reproduce:

1. Build a clay pit
2. Extract its resources until it runs out
3. Demolish it
4. Build another clay pit on the same spot

Boom! Now the resource count has reset to the full amount again. I don't know what's the point of adding a limit to deposits if you can circumvent it like this.

After you demolished the clay pit, the clay deposit re-appeared with its full original resource count.

You don't actually have to wait until you run out of resources. It also works if you only extract a little bit of clay. I believe this bug also happens with other resources (it's NOT tested, however).

**Comment by LinuxDonald** [comment]
Yeah we know about this bug. For years now. But it don't will get fixed in this version of UH. We don't have any devs that's are working on it. We porting UH over to Godot Engine.
